File: removeDuplicates.java
public class Solution {
    public int removeDuplicates(int[] nums) {
        int size = nums.length, count = 1;
        if (size == 0 || size == 1)
            return size;
        
        for (int i = 0; i < size-1; i++){
            if (nums[i] != nums[i+1])
                nums[count++] = nums[i+1];
            
        }
        
        
        return count;
    }
}

File: removeDuplicates.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) < 3:
            return len(nums)
        
        total = 1
        count = 1
        for i in range(1, len(nums)):
            count += 1
            if nums[i] != nums[i-1]:
                count = 1
            if count > 2:
                continue
            
            total += 1
            nums[total-1] = nums[i]

        
        return total
        
